api/http/handler/endpoint.go

```diff
@@ -58,7 +59,7 @@ func NewEndpointHandler(bouncer *security.RequestBouncer, authorizeEndpointManag
 type (
 	postEndpointsRequest struct {
 		Name      string `valid:"required"`
-		URL       string `valid:"required"`
```

Comment:
I'd rather introduce a boolean in `postEndpointsRequest`, something like `Local` to specify if the creation of the endpoint should rely on the platform check. This field would be required.

I don't really like the fact that the creation of a local endpoint relies on the `Name` field and an empty URL. Could you update the `api/swagger.yaml` file as well?